### DISCUSSION TITLE
How can I show a tab based on a URL anchor?

### DISCUSSION THREAD

**Discussion by bombillazo** [ant-design/ant-design#42480] [Q&A] [answered]
Lets say I have a Tabs component with keys `home`, `info`, and`user`.  How can I make a specific tab open on page load if the URL is something like `domain.com/page#info`?

**Comment by kiner-tang** [comment, marked as answer]
You can use like this:

https://codesandbox.io/s/ji-ben-antd-5-5-0-forked-d1p1h0

**Reply by bombillazo** [reply]
Perfect! The only thing I needed to change was:

```tsx
  const [tab, setTab] = useState(window.location.hash.slice(1));
```

**Reply by KnightSlayer** [reply]
its sucks for me, because i cant use default browser behavior like ctrl+click to open link i new tab. any solution without <a> tag will suck
